packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1551,6 +1561,8 @@ test('mutable liquidity triggers and interest', async t => {
     chargingPeriod: SECONDS_PER_WEEK,
     recordingPeriod: SECONDS_PER_WEEK,
   };
+  t.context.minInitialLoan = 500n;
```

Comment:
Why do all the tests have this added. Most should just use the defaults.

Reply:
The production default is on the order of 5_000_000n. I could set the default in the tests to 200n, and it wouldn't effect tests. I could set it to 500n and it would only require a few tests to be updated. Do you have a preference?